index.bs

```diff
@@ -829,6 +869,21 @@ Report Long Animation Frames {#loaf-processing-model}
     1. The user agent may set |scriptTimingInfo|'s [=script timing info/source character position=] to the character position where |callback| was defined.
 </div>
 
+<div algorithm="Report pause duration">
+    To <dfn>report pause duration</dfn> given a [=duration=] |duration|:
+    1. Let |script| be the [=running script=].
+    1. Let |settings| be |script|'s [=script/settings object=].
+    1. If |settings| is not a {{Window}}, then return.
+    1. Let |document| be |settings|'s {{Window/document}}.
+    1. If |document| is not [=fully active=] or	{{Document/hidden}}, then return.
+    1. Let |frameTimingInfo| be |document|'s [=relevant frame timing info=].
+    1. If |frameTimingInfo| is null, then return.
+    1. If |frameTimingInfo|'s [=frame timing info/pending script=] is null, then return.
+    1. Increment |frameTimingInfo|'s [=frame timing info/pending script=]'s [=script timing info=/pause duration=] by the milliseconds value of |duration|.
```

Comment:
Typo -- there's an extra `=` stopping this from linking to `pause duration`.

Reply:
Fixed